packages/browser/src/notifier.ts

```diff
@@ -59,7 +59,7 @@ export class Notifier extends BaseNotifier {
   }
 
   _instrument(opt: IInstrumentationOptions = {}) {
-    opt.console = !isDevEnv(this._opt.environment);
+    if (opt.console === undefined) opt.console = !isDevEnv(this._opt.environment);
```

Comment:
Can you please format this like so?

```suggestion
    if (opt.console === undefined) {
      opt.console = !isDevEnv(this._opt.environment);
    }
```